app/decorators/c2_version_decorator.rb

```diff
@@ -0,0 +1,75 @@
+class C2VersionDecorator < Draper::Decorator
```

Comment:
yeahhhh decoratorsss :+1: 

if there are already decorators in here using Draper, that makes total sense to continue on with. If not, I would consider using SimpleDelegator since I've probably ripped Draper out of projects more often than I've added it in 

http://hashrocket.com/blog/posts/using-simpledelegator-for-your-decorators